ferrocene/doc/release-notes/src/24.11.0.rst

```diff
@@ -1,8 +1,6 @@
 .. SPDX-License-Identifier: MIT OR Apache-2.0
    SPDX-FileCopyrightText: The Ferrocene Developers
 
-:upcoming-release:
```

Comment:
Note we can't remove this before release.

Reply:
will leave this open until release is done